### DISCUSSION TITLE
could not import matplotlib

### DISCUSSION THREAD

**Discussion by hhy2022** [idaholab/moose#20459] [Q&A Getting Started] [answered]
Hi, I am a new user of MOOSE. I was trying to use peacock to run the input because I would like to see the visualized result. My command was the one provide in the slides of 2022. '~/projects/moose/python/peacock/peacock -i step1.i' 
However, when I use this command, I got a message saying that--
'Error starting peacock: Could not import matplotlib  
'You need to use the miniconda module. Run:
'module load miniconda'
I tried to run 'module load miniconda' and it said command not found. 
I guess there might be something wrong with my peacock but I don't know how to fix it.
Besides, I use Ubuntu.
Thank you very much for any help.

**Comment by GiudGiud** [comment, marked as answer]
Hello

You are using Ubuntu on your local machine?
Did you activate the conda (mamba) environement?
Can you install matplotlib with `mamba install matplotlib` ?

Guillaume

**Reply by hhy2022** [reply]
Yes. I am using Ubuntu on local machine. I had activated conda. 
I install matplotlib successfully, and I can call peakcock successfully. 
Thank you so much for your help.

**Reply by captaink-3** [reply]
hello, i have also got a same problem saying that 'Error starting peacock: Could not import vtk
'You need to use the miniconda vtk. Run:'module load miniconda'. I have also tried **mabmba install vtk**. but it does not work.

**Reply by GiudGiud** [reply]
I think there s a libmesh-vtk package you can download. @milljm suggested creating another environment just for peacock and listed the packages. It s in another post if you care to search (getting late for me)
